### PR TITLE
Add log and set markup to zero when it less than zero

### PR DESCRIPTION
**Pull request by DaniyarAskarov**

### CODE REVIEW

File: Api/Services/Markups/MarkupService.cs [hunk 1]
```diff
@@ -1,49 +1,58 @@
...
             var policies = _markupPolicyService.Get(subject, destinationInfo);
-            
+
             // Addition markups are not applied for now
             var percentPolicies = policies
                 .Where(p => p.FunctionType == MarkupFunctionType.Percent)
                 .ToList();
 
             var percentSum = percentPolicies.Sum(p => p.Value);
             if (percentSum < 0)
```
Comment (DFSko) [inline Api/Services/Markups/MarkupService.cs:39]: i guess we need to log zero percentage too
we can work for free and don't know about it

File: Api/Services/Markups/MarkupService.cs [hunk 2]
```diff
@@ -68,7 +77,9 @@ public class MarkupService : IMarkupService
 
 
         private readonly ICurrencyRateService _currencyRateService;
-        
+
         private readonly IMarkupPolicyService _markupPolicyService;
+
```
Comment (DFSko) [inline Api/Services/Markups/MarkupService.cs:82]: unnecessary line break

File: Api/Services/Markups/MarkupService.cs [hunk 3]
```diff
@@ -68,7 +77,9 @@ public class MarkupService : IMarkupService
 
 
         private readonly ICurrencyRateService _currencyRateService;
-        
+
```
Comment (DFSko) [inline Api/Services/Markups/MarkupService.cs:80]: unnecessary line break

File: Api/Services/Markups/MarkupService.cs (hunk 1 above)
Comment (DaniyarAskarov) [reply Api/Services/Markups/MarkupService.cs:39]: Make sense

**Review by AlexeyShok — APPROVED** [review]

**Pull request merged**